packages/SwingSet/src/liveslots/collectionManager.js

```diff
@@ -781,6 +782,17 @@ export function makeCollectionManager(
     return Far('weakSetStore', weakSetStore);
   }
 
+  function assertCorrectDurabilityFlags(durable, fakeDurable) {
+    assert(
```

Comment:
I'd be inclined to spell this
```js
if (fakeDurable) {
  assert(enableFakeDurable, ..);
}
```

like you did in `makeVOM`.. sure it's logically equivalent, but I think easier to read with an `if`.